src/main/java/io/cdap/plugin/salesforce/plugin/source/streaming/SalesforcePushTopicListener.java

```diff
@@ -108,7 +108,7 @@ private BayeuxClient getClient(AuthenticatorCredentials credentials) throws Exce
 
     // Set up a Jetty HTTP client to use with CometD
     HttpClient httpClient = new HttpClient(sslContextFactory);
-    httpClient.setConnectTimeout(CONNECTION_TIMEOUT_SECONDS);
+    httpClient.setConnectTimeout(TimeUnit.SECONDS.toMillis(CONNECTION_TIMEOUT_SECONDS));
```

Comment:
Let's rename this to CONNECTION_TIMEOUT_MS and initialiaze as  CONNECTION_TIMEOUT_MS = TimeUnit.SECONDS.toMillis(110);

Reply:
Refactored the code. 
87a801f